src/mtool/ui/src/components/IbofOsOperationComponents/RunIbofOs/index.js

```diff
@@ -202,7 +210,7 @@ const RunIbofOs = props => {
 	{props.OS_Running_Status !== 'Not Running' ? (
```

Comment:
Please check the indentation of the above code.

Reply:
I have corrected the indentation. Thanks for the comment